adal/constants.py

```diff
@@ -173,6 +173,8 @@ class Saml(object):
 
     TokenTypeV1 = 'urn:oasis:names:tc:SAML:1.0:assertion'
     TokenTypeV2 = 'urn:oasis:names:tc:SAML:2.0:assertion'
+    OasisWssSaml11TokenProfile11 = "http://docs.oasis-open.org/wss/oasis-wss-saml-token-profile-1.1#SAMLV1.1"

```

Comment:
Thanks @brentschmaltz for his hint in #186 to mention these 2 new constants. Interestingly, this new constant was always used in a tag named `SecurityTokenReference` in [this doc](http://docs.oasis-open.org/wss-m/wss/v1.1.1/os/wss-SAMLTokenProfile-v1.1.1-os.html#_Toc307397288), same here in our test sample [line 80 i.e. the first highlighted line here](https://github.com/AzureAD/azure-activedirectory-library-for-python/blob/1.2.0/tests/wstrust/RSTR.xml#L80-L84). However our implementation was always extracting its line 84 i.e. the last highlighted line in the link above. Since they exist on different XPATH, I previously assumed they might be related but weren't exactly the same thing.

That being said, @abhidnya13 already got the support engineer to test out this topic branch and it works. As long as it works, I'm fine with this change too. :-)

Reply:
@abhidnya13  @rayluo does: _get_saml_grant_type(wstrust_response) do any xml normalization?

Does the xml get changed in any way? This can result in a signature validation when the token is processed by the receiver.

Reply:
Discussed with @brentschmaltz offline. For his question "Why do you even care about the token type? If the IDP accepts it, why fillter on specific types. This always leads to issues.", the explanation is: (1) our parser does return whatever saml token type as-is; (2) But later the AAD, an OAuth2 AS, would expect a grant type which are different literal strings, so we will have to do the filtering/convertion. (3) This PR is good for merge now.